doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -1407,8 +1407,10 @@ machines or containers.
    * - inf.ntw.02
      - Infrastructure
      - Network
-     - The Architecture must support fully redundant network connectivity to the Kubernetes
-       nodes, leveraging multiple network connections.
+     - The Architecture must support fully redundant network connectivity to the Kubernetes 
+       nodes. At least two physical network connections per Kubernetes node are required.
```

Comment:
as the Kubernetes node can be virtual, can we really state, "two physical network connections"? Since there is a sentence for "virtualized", suggest making this for physical Kubernetes node.
```suggestion
       nodes. At least two physical network connections are required for each physical Kubernetes node.
```